plugins/backend-trace-forwarder/src/Cardano/BM/Backend/TraceForwarder.lhs

```diff
@@ -179,19 +193,19 @@ instance (FromJSON a, ToJSON a) => IsBackend TraceForwarder a where
       Just addr -> do
         queue <- atomically $ TBQ.newTBQueue queueMaxSize
         counter <- newIORef 0
-        tfMVar <- newMVar $ TraceForwarderInternal
-                              { tfQueue            = queue
-                              , tfHandle           = Nothing
-                              , tfRemoteAddr       = addr
-                              , tfFilter           = Debug
-                              , tfDispatcher       = Nothing
-                              , tfQueueFullCounter = counter
-                              , tfGetStateDigest   = return []
-                              }
-        return $ TraceForwarder tfMVar
+        tfTVar <- newTVarIO $ TraceForwarderInternal
```

Comment:
If you insert a linebreak before `TraceForwarderInternal`, you won't have to re-indent this next time you change the call chain.

Reply:
Done.